easybuild/easyconfigs/f/flook/flook-0.8.1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,33 @@
+name = 'flook'
+version = '0.8.1'
+
+homepage = 'https://github.com/ElectronicStructureLibrary/flook'
+description = """The fortran-Lua-hook library."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/ElectronicStructureLibrary/flook/archive/']
+sources = ['v%(version)s/flook-%(version)s.tar.gz']
+patches = [
+    ('flook-%(version)s_setup.make', './setup.make'),
+    'flook-%(version)s_flook.pc.in.patch',
+]
+checksums = [
+    {'flook-0.8.1.tar.gz': 'beb15b8cb57b7a2facc7a5094326aaa877fda4fe35f8aabac023232c2e12d97e'},
+    {'flook-0.8.1_setup.make': '6df3f53faa8a8fe61534ded997c5e748d0327c13b18972fbbf49eacbda30d6e0'},
+    {'flook-0.8.1_flook.pc.in.patch': '1aa3635d83f0f9bad2ef2f9457183aefa94c8f07e66d3ef31e8897190752b42c'},
+]
+
+dependencies = [('Lua', '5.4.4')]
+
+parallel = 1
+
+local_comp_flags = 'VENDOR="gnu" FFLAGS="$FFLAGS" CFLAGS="$CFLAGS"'
+buildopts = 'liball %s' % local_comp_flags
+
+sanity_check_paths = {
+    'files': ['include/flook.mod', 'lib/libflook.a', 'lib/libflookall.a'],
+    'dirs': [],
+}
```

Comment:
Given that you've gone to the effort to create the easyblock to include the pc files, you could move the creation of the build options in there, and also construct the sanity check there (and include the missing check that ensure the pc file is actually installed). You can also do the restriction of parallelism there. That would make this easyconfig totally minimal going forward.

Reply:
Thanks for the suggestion. I have updated the easyblock and easyconfig